src/app/zap-templates/zcl/data-model/chip/scene.xml

```diff
@@ -41,121 +41,120 @@ limitations under the License.
+    <command source="client" code="0x03" name="RemoveAllScenes" response="RemoveAllScenesResponse" optional="false" cli="chip scenes rmall">
       <description>
-        Command description for RemoveAllScenes
+        Remove all scenes, corresponding to the value of the GroupID field, from its Scene Table
       </description>
-      <arg name="groupId" type="INT16U"/>
+      <arg name="groupId" type="group_id"/>
       <access op="invoke" role="manage"/>
     </command>
     
-    <command source="client" code="0x04" name="StoreScene" response="StoreSceneResponse" optional="false" cli="zcl scenes store">
+    <command source="client" code="0x04" name="StoreScene" response="StoreSceneResponse" optional="false" cli="chip scenes store">
       <description>
-        Command description for StoreScene
+        Adds the scene entry into its Scene Table along with all extension field sets corresponding to the current state of other clusters on the same endpoint
       </description>
-      <arg name="groupId" type="INT16U"/>
+      <arg name="groupId" type="group_id"/>
       <arg name="sceneId" type="INT8U"/>
       <access op="invoke" role="manage"/>
     </command>
     
-    <command source="client" code="0x05" name="RecallScene" optional="false" cli="zcl scenes recall">
+    <command source="client" code="0x05" name="RecallScene" optional="false" cli="chip scenes recall">
       <description>
-        Command description for RecallScene
+        Set the attributes and corresponding state for each other cluster implemented on the endpoint accordingly to the resquested scene entry in the Scene Table
       </description>
-      <arg name="groupId" type="INT16U"/>
+      <arg name="groupId" type="group_id"/>
       <arg name="sceneId" type="INT8U"/>
-      <arg name="transitionTime" type="INT16U" introducedIn="zcl-7.0-07-5123-07" optional="1"/>
+      <arg name="transitionTime" type="INT16U" optional="1"/>
     </command>
    
-    <command source="client" code="0x06" name="GetSceneMembership" response="GetSceneMembershipResponse" optional="false" cli="zcl scenes get">
+    <command source="client" code="0x06" name="GetSceneMembership" response="GetSceneMembershipResponse" optional="false" cli="chip scenes get">
       <description>
-        Command description for GetSceneMembership
+        Get an unused scene identifier when no commissioning tool is in the network, or for a commissioning tool to get the used scene identifiers within a certain group 
       </description>
-      <arg name="groupId" type="INT16U"/>
+      <arg name="groupId" type="group_id"/>
     </command>
 
-    <command source="client" code="0x40" name="EnhancedAddScene" response="EnhancedAddSceneResponse" optional="true" introducedIn="zll-1.0-11-0037-10" cli="zcl scenes eadd">
+    <command source="client" code="0x40" name="EnhancedAddScene" response="EnhancedAddSceneResponse" optional="true" cli="chip scenes eadd">
```

Comment:
Typo with `eadd` ?

Reply:
it left the cli command as it was before but with chip instead of zcl. `eadd` is for Enhanced add I assume